test/lib/www/load-images.html

```diff
@@ -0,0 +1,6 @@
+<html>
+<body>
+    <img src="logo.png"/>
+    <img src="phantomjs.png"/>
```

Comment:
Yeah that file could probably stand to have the dust blown off it a little.  No worries.